tests/org.eclipse.reddeer.eclipse.test/src/org/eclipse/reddeer/eclipse/test/wst/server/ui/view/ServersViewTestCase.java

```diff
@@ -80,9 +90,55 @@ protected static void importProjects(){
 		wizard.open();
 
 		WizardProjectsImportPage wizardPage = new WizardProjectsImportPage(wizard);
-		wizardPage.setArchiveFile(ZIP_FILE.getAbsolutePath());
+		wizardPage.setArchiveFile(PROJECTS_ZIP_FILE.getAbsolutePath());
 		wizardPage.selectProjects("server-project", "server-project-2", "server-project-3");
 
 		wizard.finish();
 	}
+	
+	protected static void importMultimodularProject(){
+		MavenImportWizard wizard = new MavenImportWizard();	
```

Comment:
@zcervink I suppose it would be better to use the same approach as it is used in importProjects(), then you would not need to play with zip archive.

Reply:
Done.